recipes/pynput/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pynput" %}
+{% set version = "1.6.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 68c1863d6a1520b44b6a915e866cbfa1b8d127aef9289f25183c93e28ee5049a
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - six
+    - setuptools-lint >=0.5
+    - sphinx 
+    - wrapt ==1.11.2
+    - pyobjc-framework-quartz  # [osx]
```

Comment:
Are there required to build the package or can they be in `run` only?

Reply:
I've checked each and few aren't necessary for `host`.